roles/aci/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This role contains tasks specific to aci
+
+  - name: Install docker-py as a workaround for Ansible issue
```

Comment:
Right. What I'm saying is all we do is extract the version information 
for installing the latest version of docker.

If we can do that with a shell command, a lot of this no longer becomes 
necessary. Can we do that instead, or is there something I'm missing 
here?

On 6 Dec 2015, at 13:48, jojimt wrote:

>> @@ -0,0 +1,23 @@
>> +---
>> +# This role contains tasks specific to aci
>> +
>> +  - name: Install docker-py as a workaround for Ansible issue
>
> The issue is that docker-py is needed for docker. Without it, docker 
> module returns random errors. Found this from the github discussion on 
> the topic.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/ansible/pull/3/files#r46776631